app/drivers/nomad/mac_virtualizer.go

```diff
@@ -408,3 +411,11 @@ nc -zv $(/opt/homebrew/bin/tart ip %s) 9079
 func (mv *MacVirtualizer) GetHealthCheckPort(portLabel string) string {
 	return fmt.Sprint(lehelper.LiteEnginePort)
 }
+
+func (mv *MacVirtualizer) GetInitJobTimeout(vmImageConfig types.VMImageConfig) time.Duration {
+	// This makes sure that only customer specific images have a higher init timeout
+	if strings.HasPrefix(vmImageConfig.ImageName, harnessImageName) {
```

Comment:
@raghavharness I would have relying on prefix. We need this as another configuration passed from ci-mgr.